binder/postBuild

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# exit on error
+set -e
+
+# use code dir for code
+mkdir code
+cd code
+
+# get the source code
+git clone https://github.com/OSGeo/grass.git
```

Comment:
This seems to allway build the official master branch. So, it wil not work for testing PRs, right?

I tried to find some info on how to solve this but did not come across a solution that is known to work, but you could try if it is possible in the postbuild script to:

1 fetch teh url of the current repo:
`git config --get remote.origin.url`

and
2. use that to clone the code from which GRASS is supposed to be compiled...

But maybe @wenzeslaus has some more clear ideas about this...

Reply:
Just removing all the `cd`s, `mkdir`s and clone should be enough. Binder actually puts the repo content to what is the current directory when the Docker container starts. This makes it simply use always the current code whatever is the the current one, master branch in upstream, branch on a fork, ...

You can test that by going to Binder, the URL is (obtained by filling out mybinder.org):

<https://mybinder.org/v2/gh/chaedri/grass/binder>

@ninsbl I'm not sure what you are aiming at, but if I understand correctly, if you can do `git config --get remote.origin.url`, you probably have the repo clone already, so you don't need to do it at all.

Reply:
@wenzeslaus , thanks great, I was not aware of how this is handled in binder. I just saw the line that clones the official repo which whould not result in the behavior we discused. So, if compiling from current directory works, that is great and please disregard my suggestion of `git config`...